xc7/utils/splitter/conn_splitter.py

```diff
@@ -0,0 +1,505 @@
+#!/usr/bin/env python3
```

Comment:
How was the new tileconn tested?  Have you considered an approach that wouldn't required writing a new tileconn.json?

Reply:
For "testing" I've written an utility that draws the FPGA grid along with connection using data in `tilegrid.json` and `tileconn.json`. I can visually examine if connection on the splitted grid are ok.

It is still WIP. For now there is one connection case which has not been converted and it is when carry chain cross clock region boundary.